### PR TITLE
Adds functional tests for scanner component of service

### PR DESCRIPTION
**Pull request by navidshaikh**
Runs a minimal container build and tests scanner component functionalities.

### CODE REVIEW

**Comment by navidshaikh** [comment]
#dotests

**Comment by navidshaikh** [comment]
#dotests

**Comment by navidshaikh** [comment]
#dotests

**Comment by navidshaikh** [comment]
#dotests

**Comment by navidshaikh** [comment]
The tests are through, I have updated the pipeline scanner per its current behavior [here](https://github.com/CentOS/container-pipeline-service/pull/434/files#diff-0ab7f6c63253da040b4d589247b2aec3R155). PTAL

File: ci/tests/test_01_functional/test_03_scanner/__init__.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+                   "python container_pipeline/pipeline.py {3}").format(
+            workspace_dir, self.repo_url, self.test_tag, args)
+
+        # we need to trigger build on jenkins_slave node
+        # figure out jenkins_slave hostname to run command upon
+        # self.hosts variable is defined in base.py
+        jenkins_slave = self.hosts["jenkins_slave"]["host"]
+
+        print "Starting a test build with command: \n%s" % command
+        print self.run_cmd(command, host=jenkins_slave)
```
Comment (bamachrn) [inline ci/tests/test_01_functional/test_03_scanner/__init__.py]: we are manually running the command in jenkins slave. So we do not need the jenkins job entry for this.

**Pull request merged**